internal/jennies/java/registry.go

```diff
@@ -97,15 +114,22 @@ func (jenny Registry) renderRegistry(context languages.Context) (string, error)
 		return panelSchemas[i].Identifier < panelSchemas[j].Identifier
 	})
 
+	if jenny.config.generateConverters {
+		imports.Add("Panel", "dashboard")
```

Comment:
I'm surprised to see this here: the runtime shouldn't assume that we're generating code related to dashboards/panels